docs/generic-bots/monkey-bot.mdx

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: 'Monkey Bot'
+---
+
+# Monkey Bot
+
+![Monkey Bot Image](img/monkey-bot.png)
+
+The Regression Games SDK has a built-in **Monkey Bot** that automatically determines the applicable keyboard and mouse inputs for the game under test and randomly performs them as the game is running. The SDK also offers the capability to configure the actions taken by the bot to control the functionality that is exercised by the bot.
```

Comment:
```suggestion
The Regression Games SDK has a built-in **Monkey Bot** that automatically determines the applicable keyboard and mouse inputs for the game being tested and randomly performs those actions as the game is running. The SDK allows you to select which of the actions are enabled in order to control/limit the functionality that is exercised by the bot.
```

Reply:
Doesn't need to be my exact wording in the suggestion, but we need to reword this section to remove some wordiness and provide clarity.